src/Core/Language/Pack/Loader/RemoteLanguagePackLoader.php

```diff
@@ -34,35 +36,52 @@
 final class RemoteLanguagePackLoader implements LanguagePackLoaderInterface
 {
     /**
-     * The link from which available languages are retrieved.
+     * The languages repository base path
+     *
+     * @TODO : duplicated from {prestashop_repository}/app/config/config.yml;
+     *         used for initial auto-install (no symfony container available)
      */
-    public const PACK_LINK = 'http://i18n.prestashop.com/translations/%ps_version%/available_languages.json';
+    private const LANG_REPOSITORY_BASE_PATH = 'http://i18n.prestashop-project.org';
 
     /**
      * @var Version
      */
-    private $version;
+    private Version $version;
+
+    /**
+     * @var string represents the language repository base path
+     * @TODO : change type for GuzzleHttp\Client
```

Comment:
I don't see why this should be switched to guzzle client?

If you plan on replacing file_get_contents with a guzzle client someday why not But then the todo should be in the `getLanguagePackList` function not here where it's not related